docs/architecture/anti-entropy/read-repair.rst

```diff
@@ -39,38 +39,8 @@ See the appendices below for the detailed flow.
 
 Probabilistic Read Repair
 ^^^^^^^^^^^^^^^^^^^^^^^^^
-See the appendices below for the detailed flow. 
-
-Another read repair mechanism is **probabilistic** read repair, which works as follows:
-
-1. The coordinator checks the value of the ``read_repair_chance`` or ``dclocal_read_repair_chance``.   If the probability of a read repair is triggered, it will start the read repair across the entire cluster, or inside the DC, respectively.
-
-.. note::
-   
-   These two settings are configured in ``scylla.yaml``.
-   In recent Scylla versions (starting from 4.2.0) default value of both parameters is zero, making Probabilistic Read Repair **off by default**, with a plan to `remove it in the future <https://github.com/scylladb/scylla/issues/3502>`_
-
-2. If the probability of a read repair is triggered, the coordinator sends data request to one of the nodes, digest request to all of the remaining nodes.
-
-.. image:: 5_read_repair.png
-
-3. The coordinator responds to the client as soon as there are enough replies to achieve consistency level, and there is a match.
 
-.. raw:: html
-
-	&nbsp;
-   
-4. After ALL replicas responses to the coordinator (which may have already responded to the client), the coordinator will compare all responses; and if required (there is a mismatch), the coordinator will ask for the  data from all replicas, run reconciliation, and send deltas to any out-of-date replica node(s) - in the background.
-
-.. raw:: html
-
-	&nbsp;
-
-5. If a query’s consistency level is more than 1, and a data mismatch is found before consistency level is reached, then the read repair will run in the foreground. **All replica nodes are updated before the data is returned to the client.** However, if data mismatch  is found *after* a query’s consistency level is achieved, the update happens in the background, as mentioned above.
-
-The probability of this kind of read repair occurring on a read request is based on the :code:`read_repair_chance`. This :doc:`setting </cql/ddl/>`, provided when a table is created, can apply across data centers.  The setting specifies the probability that extra nodes (in addition to those required by the consistency level) are queried for the purpose of read repairs.
-
-A similar :doc:`setting </cql/ddl/>`, :code:`dclocal_read_repair_chance`,  applies only to nodes within the local data center.  You can use this setting when conserving traffic between data centers is important.
+Probabilistic read repair configured using ``read_repair_chance`` and ``dclocal_read_repair_chance`` in ``scylla.yaml`` is removed in Scylla 6.1.
```

Comment:
ScyllaDB 6.0, no?

@annastuchlik I recall you have some guidelines on whether we should mention things like "removed in version X", as we now have per-version documentation anyway.

Reply:
@denesb hi Botond, this magic (version) number came from #3502, where the milestone was marked 6.1. so i blindly use it. but i agree with you, before GA, the first release version in the 6.x branch should always be 6.0.

Reply:
@annastuchlik hi Anna, thank you for pointing this out. dropped.

Reply:
@dani-tweig hi Dani, how about adding a dedicated github label for the release-note worthy changes?
@tzach hi Tzach, we could have a `PendingReleaseNotes` file in the source repo, and require developer to document the change in this file. back when working on Ceph, we have https://github.com/ceph/ceph/blob/main/PendingReleaseNotes . it'd be more efficient to document these changes this way, what do you think?